src/Controller/MoocController.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace AppBundle\Controller;
+
+use AppBundle\Entity\Mooc\AttachmentFile;
+use League\Flysystem\FilesystemInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\ResponseHeaderBag;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration as SFConf;
+
+/**
+ * @SFConf\Route("/mooc")
```

Comment:
CS: please delete `SFConf` alias

Reply:
Why ? it's used multiple time

Reply:
Simply because we never did that on the project and that's not a defacto standard like `@ORM`.

Reply:
Then we have 10 things used in same namespace, it will be better to have 100 use than have an alias ?